src/javascript/static/pages/signup.js

```diff
@@ -0,0 +1,82 @@
+const BinarySocket   = require('../../app/base/socket');
+const FormManager    = require('../../app/common/form_manager');
+const getElementById = require('../../_common/common_functions').getElementById;
+const localize       = require('../../_common/localize').localize;
+const Login          = require('../../_common/base/login');
+
+const Signup = (() => {
+    let clients_country,
+        $google_btn,
+        $login_btn;
+
+    const form_id = '#signup_form';
+
+    const onLoad = () => {
+        getElementById('footer').setVisibility(0); // always hide footer in this page
+
+        $google_btn = $('#google-signup');
+        $login_btn  = $('#login');
```

Comment:
some elements are stored, others are called on the fly, e.g. `$('.error-msg')`, `$(form_id)`, `$('#verify_email')`, it would be better to use the same standard